source/MaterialXTest/CMakeLists.txt

```diff
@@ -126,6 +126,11 @@ endif()
 add_custom_command(TARGET MaterialXTest POST_BUILD
 	COMMAND ${CMAKE_COMMAND} -E copy_directory
 	${CMAKE_CURRENT_SOURCE_DIR}/../../libraries ${CMAKE_BINARY_DIR}/bin/libraries)
+if(MATERIALX_BUILD_GEN_ESSL)
```

Comment:
What difference does this change make?

Reply:
This will copy over the files (target definition, potential essl pbrlib/stdlib overrids) from the MaterialXGenEssl/libraries folder into the bin/libraries dir so that when you start MaterialXView it has access to those.